lms/djangoapps/grades/tests/test_new.py

```diff
@@ -158,9 +160,9 @@ def test_subsection_grade_feature_gating(self, feature_flag, course_setting):
         with patch(
             'lms.djangoapps.grades.models.PersistentSubsectionGrade.read_grade'
         ) as mock_read_saved_grade:
-            with patch.dict(settings.FEATURES, {'ENABLE_SUBSECTION_GRADES_SAVED': feature_flag}):
-                with patch.object(self.course, 'enable_subsection_grades_saved', new=course_setting):
-                    self.subsection_grade_factory.create(self.sequence, self.course_structure, self.course)
+            PersistentGradesEnabledFlag.objects.create(enabled=feature_flag)
+            CoursePersistentGradesFlag.objects.create(course_id=self.course.id, enabled=course_setting)
```

Comment:
Can you create a helper in `tests/utils.py` for this?  Seems like we'll need it a few places - and more as we add more test coverage.